### PR TITLE
Add multisignature commands to the test script - Closes #3134

### LINKED ISSUE #3134: Add lisk-commander commands to the test script for multisignature transactions
### Description

The test script should register a multisignature account and also send one transfer transaction so that we can test multisignature transactions.

### Motivation

We would like to have e2e testing for multisignature registration and other transactions. 

### Acceptance Criteria

The test script should: 

1. Create at least one multisignature account
2. Create at least one multisignature transaction

### PR DESCRIPTION
**Pull request by UsamaHameed**
### What was the problem?
This PR adds commands to our script for e2e tests for creating and sending multisignature transactions.

### How was it solved?
Added new lisk-commander commands.

### How was it tested?
Looked at the responses from `/accounts` and `transactions` endpoints.

### CODE REVIEW

**Review by reyraa — DISMISSED** [review]
Perfetta!

File: test/create-test-transactions.sh [hunk 1]
```diff
@@ -3,21 +3,56 @@
 lisk config:set api.nodes http://localhost:4000
 
 NETWORKIDENTIFIER="93d00fe5be70d90e7ae247936a2e7d83b50809c79b73fa14285f02c842348b3e"
-PASSPHRASE="peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready"
+GENESIS_NONCE=103
+MULTISIG_CANDIATE_NONCE=0
+# genesis
+PASSPHRASE1="peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready"
```
Comment (isalga) [inline test/create-test-transactions.sh:9]:
```suggestion
PASSPHRASE_GENESIS="...
```

File: test/create-test-transactions.sh [hunk 2]
```diff
@@ -3,21 +3,56 @@
...
+OPTIONALKEY1="0fe9a3f1a21b5530f27f87a414b549e79a940bf24fdf2b2f05e7f22aeeecc86a"
 
-NONCE=103
+# delegate
+PASSPHRASE2="recipe bomb asset salon coil symbol tiger engine assist pact pumpkin visit"
```
Comment (isalga) [inline test/create-test-transactions.sh:13]:
```suggestion
PASSPHRASE_DELEGATE="...
```

File: test/create-test-transactions.sh [hunk 3]
```diff
@@ -3,21 +3,56 @@
...
+TRANSACTION1=$(lisk transaction:create:multisignature "$MULTISIG_CANDIATE_NONCE" 0.1 --number-of-signatures=2 --optional-key="$OPTIONALKEY1" --optional-key="$OPTIONALKEY2" --passphrase="$PASSPHRASE3" --member-passphrase="$PASSPHRASE1" --member-passphrase="$PASSPHRASE2" --networkIdentifier="$NETWORKIDENTIFIER")
+
+# sign the registration transaction with genesis account
+TRANSACTION2=$(lisk transaction:sign "$TRANSACTION1" --optional-key="$OPTIONALKEY1" --optional-key="$OPTIONALKEY2" --passphrase="$PASSPHRASE1" --networkIdentifier="$NETWORKIDENTIFIER")
+
+# broadcast the tx
+lisk transaction:broadcast "$TRANSACTION2"
+
+# wait for the account registration tx to be included in the blockchain
+sleep 15
```
Comment (isalga) [inline test/create-test-transactions.sh:47]: Would be nice to have this code wrapped on a function so it becomes more readable. I suggest `registerMultisignatureAccount()`

File: test/create-test-transactions.sh [hunk 4]
```diff
@@ -3,21 +3,56 @@
 lisk config:set api.nodes http://localhost:4000
 
 NETWORKIDENTIFIER="93d00fe5be70d90e7ae247936a2e7d83b50809c79b73fa14285f02c842348b3e"
-PASSPHRASE="peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready"
+GENESIS_NONCE=103
+MULTISIG_CANDIATE_NONCE=0
+# genesis
+PASSPHRASE1="peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready"
+OPTIONALKEY1="0fe9a3f1a21b5530f27f87a414b549e79a940bf24fdf2b2f05e7f22aeeecc86a"
```
Comment (isalga) [inline test/create-test-transactions.sh:10]: If this is the optional key from genesis I suggest to rename it for `OPTIONALKEY_GENESIS`

File: test/create-test-transactions.sh [hunk 5]
```diff
@@ -3,21 +3,56 @@
...
 
-NONCE=103
+# delegate
+PASSPHRASE2="recipe bomb asset salon coil symbol tiger engine assist pact pumpkin visit"
+OPTIONALKEY2="86499879448d1b0215d59cbf078836e3d7d9d2782d56a2274a568761bff36f19"
```
Comment (isalga) [inline test/create-test-transactions.sh:14]: same

File: test/create-test-transactions.sh [hunk 6]
```diff
@@ -3,21 +3,56 @@
...
 
+# mutisig candidate passphrase
+PASSPHRASE3="approve tribe main deposit luxury obtain knock problem pulse claw social select"
```
Comment (isalga) [inline test/create-test-transactions.sh:17]:
```suggestion
PASSPHRASE_MULTISIGN_CANDIDATE="...
```

File: test/create-test-transactions.sh [hunk 7]
```diff
@@ -3,21 +3,56 @@
...
+TRANSACTION3=$(lisk transaction:create:transfer 1 0.1 1 5932438298200837883L --networkIdentifier="$NETWORKIDENTIFIER" --passphrase="$PASSPHRASE3")
+
+# sign using one member
+TRANSACTION4=$(lisk transaction:sign "$TRANSACTION3" --optional-key="$OPTIONALKEY1" --optional-key="$OPTIONALKEY2" --passphrase="$PASSPHRASE1" --networkIdentifier="$NETWORKIDENTIFIER")
+
+# sign using another member
+TRANSACTION5=$(lisk transaction:sign "$TRANSACTION4" --optional-key="$OPTIONALKEY1" --optional-key="$OPTIONALKEY2" --passphrase="$PASSPHRASE2" --networkIdentifier="$NETWORKIDENTIFIER")
 
-# docker exec -t docker_db_1 pg_dump -U lisk lisk > ./dev_blockchain.db
+lisk transaction:broadcast "$TRANSACTION5"
```
Comment (isalga) [inline test/create-test-transactions.sh:58]: Same. I suggest `multisignatureTransaction()`

**Review by isalga — CHANGES_REQUESTED** [review]

File: test/create-test-transactions.sh (hunk 3 above)
Comment (UsamaHameed) [reply test/create-test-transactions.sh:47]: We can do this once we use those lines more than once. Right now, it is difficult to parametrise things there and honestly, would feel a bit over engineered. I have added comments explaining what the next command does to make things readable.

File: test/create-test-transactions.sh (hunk 7 above)
Comment (UsamaHameed) [reply test/create-test-transactions.sh:55]: https://github.com/LiskHQ/lisk-desktop/pull/3133#discussion_r514141941

**Review by reyraa — APPROVED** [review]

**Review by isalga — APPROVED** [review]

**Pull request merged**